lib/features/login/domain/exceptions/login_exception.dart

```diff
@@ -1,4 +1,21 @@
 
+import 'package:flutter/services.dart';
+
 class NotFoundDataResourceRecordException implements Exception {}
 
-class NotFoundUrlException implements Exception {}
\ No newline at end of file
+class NotFoundUrlException implements Exception {}
+
+class NoSuitableBrowserForOIDCException implements Exception {
+
+  static const noBrowserAvailableCode = 'no_browser_available';
+
+  static bool verifyNoSuitableBrowserOIDC(dynamic exception) {
+    if (exception is PlatformException) {
+      if (exception.code == noBrowserAvailableCode) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
I don't like writing like that. Each exception class should only serve a single error. We shouldn't deal with logic here.

Reply:
quite not get your point, it is exception from platform, we should convert it back to business exception